bin/meshtasticd.service

```diff
@@ -5,10 +5,11 @@ StartLimitInterval=200
 StartLimitBurst=5
 
 [Service]
-User=root
-Group=root
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+User=meshtasticd
+Group=meshtasticd
 Type=simple
-ExecStart=/usr/sbin/meshtasticd
+ExecStart=/usr/bin/meshtasticd
```

Comment:
Don't we need the fsdir flag here, to point to the new virtual filesystem location?

Reply:
I see what you did there.